### DISCUSSION TITLE
Support for `waymo monocular` task

### DISCUSSION THREAD

**Discussion by Xiangxu-0103** [open-mmlab/mmdetection3d#1919] [General]
Hi, for `tfrecrod` files under `testing_3d_camera_only_detection`, it will cause an error that `'NoneType' object has no attribute 'data'` when saving lidar points. It seems that the `tfrecord` files do not contain point cloud data.

**Comment by Tai-Wang** [comment]
Thanks for your kind reminder. We will check this problem and fix it if necessary ASAP.
